webapp/config/packages/doctrine.yaml

```diff
@@ -19,14 +19,17 @@ doctrine:
         mapping_types:
             enum: string
         schema_filter: ~^(?!sessions)~
+        # IMPORTANT: You MUST configure your server version,
```

Comment:
s/server version/mysql major server version/ ?

Reply:
This is what symfony adds in a recipe. I’m reluctant to change it, since it will make future recipe updates conflict

Reply:
hmmm, ok